src/ucp/wireup/wireup.c

```diff
@@ -1004,6 +1004,19 @@ ucp_wireup_ep_lane_set_next_ep(ucp_ep_h ep, ucp_lane_index_t lane,
                               ucp_ep_get_rsc_index(ep, lane));
 }
 
+static int ucp_wireup_activate_worker_iface(ucp_worker_iface_t *wiface,
```

Comment:
need to name this function as predicate
maybe "ucp_wireup_should_activate_wiface"